doc/news/split_changes_h.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2016 by the deal.II authors
+##
+## This file is part of the deal.II library.
+##
+## The deal.II library is free software; you can use it, redistribute
+## it, and/or modify it under the terms of the GNU Lesser General
+## Public License as published by the Free Software Foundation; either
+## version 2.1 of the License, or (at your option) any later version.
+## The full text of the license can be found in the file LICENSE at
+## the top level of the deal.II distribution.
+##
+## ---------------------------------------------------------------------
+
+#
+# This script splits the previously used into contributions in the folders:
```

Comment:
still, i would not commit it. I just don't see a reason why should it be a part of the Git history.  I would just split the current `changes.h` (without deleting it!) into lots of smaller once using these scripts and make sure that re-running generation does not show any missing entries.